### PR TITLE
refactor: move DayOfWeek to render props components

### PR DESCRIPTION
**Pull request by havenchyk**
### Description

Move DayOfWeek under RenderPropsComponents, add default render function to reduce complexity

### Review

- [x] Verify that all classes affected by the changes have **good** top-level documentation.
- [ ] Verify that new code has been covered with specs (and features if applicable).
- [ ] Verify that a corresponding meaningful changelog entry has been added (`* Add Github pull request template`), for this change if applicable.
- [ ] Test manually.
- [ ] Get two :+1: from code review.

### Pre-merge checklist

- [ ] The PR relates to a single subject with a clear title and description in grammatically correct, complete sentences.
- [ ] Verify that all new dependencies are included in `package.json`.
- [ ] Double check the quality of [commit messages](http://chris.beams.io/posts/git-commit/).
- [ ] Squash related commits together.
- [ ] Verify the branch name starts with `NNNN` where `NNNN` is the issue number.

### CODE REVIEW

File: src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
+import classnames from 'classnames'
+
+export type Props = {
```
Comment (PeterMK85) [inline src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx]: Not required to be exported

File: src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react'
```
Comment (PeterMK85) [inline src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx]: I would also add a simple test for `RenderPropsComponents/DayOfWeek/DayOfWeek.tsx` to cover the main functionality (class addition basically)

Comment (havenchyk) [reply src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx]: Well, it’s kind of testing implementation details, but okay, I’ll add a snapshot test

Comment (havenchyk) [reply src/RenderPropsComponents/DayOfWeek/DayOfWeek.tsx]: done

File: src/RenderPropsComponents/DayOfWeek/test.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react'
+import React, { ComponentProps } from 'react'
+
+import DayOfWeek from './DayOfWeek'
+
+const renderDayOfWeek = (props: Partial<ComponentProps<typeof DayOfWeek>> = {}) => {
+  const baseProps = {
+    blockClassName: 'example-class-name',
+    day: '2015-12-12',
```
Comment (havenchyk) [inline src/RenderPropsComponents/DayOfWeek/test.tsx]: @PeterMK85 not sure it's a correct string

**Comment by havenchyk** [comment]
@PeterMK85 ping

**Pull request merged**